prost-derive/Cargo.toml

```diff
@@ -14,7 +14,8 @@ proc-macro = true
 
 [dependencies]
 anyhow = "1.0.1"
-itertools = ">=0.10.1, <=0.13"
+heck = "0.4.1"
```

Comment:
0.5 was released 9 months ago. Any reason to use a version from 2 years ago?

Reply:
Not sure anymore but I guess this PR or at least my first local impl predated this commit https://github.com/tokio-rs/prost/commit/2b6140cf6fe899e9817816bae97d19fe8e0eab60.

But yeh, we can certainly bump this.